11-node-frameworks/classworks/Agatha Rudko/index.js

```diff
@@ -0,0 +1,56 @@
+const serve = require('koa-static');
+const Koa = require('koa');
+const sessionss = require('koa-generic-session');
+const bodyParser = require('koa-bodyparser');
+const Router = require('@koa/router');
+const joi = require('joi');
+const router = new Router();
+
+const app = new Koa();
+let session = sessionss;
+let formInputs = {};
+const schem = joi.object({
+  name: joi.string().required(),
+  surname: joi.string().min(3),
+});
+
+function get(ctx) {
+  session.count = session.count || 0;
+  session.count++;
+  ctx.body = session.count;
+}
+
+function remove(ctx) {
+  session.count = 0;
+  ctx.body = session.count;
```

Comment:
Please, Use JSON for each response

Reply:
Still valid point. Each response to a client should contain JSON as body of HTTP request

Reply:
Apply this rule to every response in the code